### ISSUE TITLE
bug in gl_draw() on mac OS

### ISSUE THREAD

**Issue by vindar** [fltk/fltk#38]
There seems to be a problem in `gl_draw()` on mac OS only. 

Unlike Windows version which uses `glDrawPixels()` to draw the text., the Apple version use textures which are cached into using a `gl_texture_fifo` class. There is a problem with this caching procedure when the text contains spaces. 

The following program illustrates the problem: it correctly prints number 0 to 119 on Windows but does not on mac OS. 
```
#include <FL/Fl.H>
#include <FL/Fl_Gl_Window.H>
#include <FL/gl.h>
#include <string>
#include <sstream>

class MyGlWindow : public Fl_Gl_Window {
	void draw() {
		if (!valid()) {
			valid(1);
			glClearColor(0.0, 0.0, 0.0, 0.0);
			gl_font(1, 12);
			glColor3f(1.0f, 1.0f, 1.0f);
			glDisable(GL_DEPTH_TEST);
		}
		glClear(GL_COLOR_BUFFER_BIT);
		for (int j = 0; j < 12; j++)
			for (int i = 0; i < 10; i++) {
				std::ostringstream oss;
				oss << " x " << (i + 10 * j); /// <- no bug without the " = "...
				gl_draw(oss.str().c_str(), oss.str().size(), -1.0f + 0.2f*i, -1.0f + +0.15f*j + 0.1);
			}
	}

public:
	MyGlWindow(int X, int Y, int W, int H, const char*L = 0) : Fl_Gl_Window(X, Y, W, H, L) {
	}
};

int main() {
	Fl_Window win(600, 400);
	MyGlWindow mygl(0, 0, win.w(), win.h());
	win.show();
	return(Fl::run());
}
```

**Comment by Albrecht-S** [comment]
Please tell us the FLTK version you are using.

I believe that this or a similar bug has been fixed in version 1.3.5 (not entirely sure though) and/or in 1.4.x (Git master branch).

If you were using an older version, can you please try 1.3.5 from the download page or 1.4.x from Git?

**Comment by vindar** [comment]
The bug is for 1.3.5. I just checked and indeed it is fixed in 1.4.
I did not realize that support for 1.3.5 had ended. Then I guess there is nothing to fix !

**Comment by Albrecht-S** [comment]
Thanks for the quick confirmation.
